setup-buildengine.sh

```diff
@@ -288,9 +281,9 @@ packVM() {
     VBoxManage controlvm "$OPT_VM" poweroff 2>/dev/null
 
     # remove target archive files
-    rm -f $INSTALL_PATH/$OPT_TOOLING
```

Comment:
We still need to remove the tooling archive, it's not one of the targets

Reply:
Oh thanks I deleted one line to much.